packages/tocco-ui/src/EditableValue/typeEditors/NumberEdit.js

```diff
@@ -16,20 +16,28 @@ export const calculateMaxValue = (prePointDigits, postPointDigits, maxValue) =>
   return maxValue
 }
 
-export const isAllowedValue = (prePointDigits, postPointDigits, minValue, maxValue) => ({floatValue}) => {
-  const calculatedMaxValue = calculateMaxValue(prePointDigits, postPointDigits, maxValue)
+const getPreDecimalPositions = number => Math.ceil(Math.log10(number + 1))
```

Comment:
change seems to work. 
but now when the minimum is 10 and someone enters "1" nothing indicates that this value is not set and therefore not saved unless its a mandatory field.
Maybe if the input is shorter than the minimum value, the input should be cleared on focus change.